src/test/java/com/redhat/devtools/lsp4ij/features/formatting/TypeScriptEditorImprovementsTest.java

```diff
@@ -361,22 +376,22 @@ public void testStatementTerminatorInBlockComment() {
     // This exercises LSPEditorImprovementsEnterBetweenBracesHandler
 
     public void testEnterBetweenBraces_spaces() {
-        testEnterBetweenSpaces(false);
+        testEnterBetweenBraces(false);
```

Comment:
Typo fix in an unrelated test. Should have been `EnterBetweenBraces` and not `EnterBetweenSpaces`.